bindings/wasm/package.json

```
@@ -18,12 +18,13 @@
     "build:examples": "webpack --config ./examples/webpack.config.js --mode=production",
     "build": "npm run build:web && npm run build:nodejs && npm run build:docs",
     "build-dev": "npm run build-dev:web && npm run build-dev:nodejs && npm run build:docs",
-    "pretest": "npm run build:nodejs",
     "prepublishOnly": "npm run build",
     "serve": "webpack serve",
     "example:node": "node examples/dist/node.js",
     "example:browser": "http-server ./examples/dist/ -c-1 -o ",
+    "test": "concurrently -g --timings \"npm:test:node\" \"npm:test:browser:parallel\" \"npm:test:readme\" \"npm:test:unit\"",
```

Nit: I think the unit tests and examples should still be separate steps in the `build-and-test-wasm` job. If the unit tests fail there is no point in running the examples, and they're relatively fast even when recompiling for the test profile. Optional since it doesn't matter that much.

```suggestion
    "test:examples": "concurrently -g --timings \"npm:test:node\" \"npm:test:browser:parallel\" \"npm:test:readme\"",
```

---

handled it inline in the npm command